src/components/common/Forms/RegisterForm/RegisterForm.jsx

```diff
@@ -189,16 +176,28 @@ export const RegisterForm = () => {
                             
 
                             <CheckboxContainer>
-                                <div onClick={handleToggleCheck}>
-                                    {
-                                        isChecked
-                                            ? <Icon name='checked' size={24} />
-                                            : <Icon name='unchecked' size={24} />
-                                    }
-                                </div>
-                                <ConfirmationText>
-                                    By checking this box, you are creating an account and you agree to the <PolicyLink target='_blank' to='/conditions'>Terms & Conditions</PolicyLink> and <PolicyLink target='_blank' to='/policy'>Privacy Policy</PolicyLink>.
-                                </ConfirmationText>
+                                <label>
+                                    <Checkbox>
+                                        <Field
+                                            name='confirmation'
+                                            type='checkbox'
+                                            onClick={handleToggleCheck}
+                                            value={isChecked}
+                                            style={{display: 'none'}}
```

Comment:
Why `display: none`? Just remove this field from the code :)

Reply:
This field is necessary for Formik lib. 

"<Field /> will automagically hook up inputs to Formik. It uses the name attribute to match up with Formik state. <Field /> will default to an HTML <input /> element."

I get state from this.